public/lib/invertedIndexTests.js

```diff
@@ -0,0 +1,113 @@
+
```

Comment:
It's a good practice for your test to be in one place --- 
--- one of your test files is in jasmine-standalone folder and now another one is in public/lib folder
You might want to create a test folder that will contain both files